mnist-keras/README.md

```diff
@@ -26,16 +39,17 @@ docker build . -t mnist-client:latest
 
 2. Start a client (edit the path of the volume mounts to provide the absolute path to your local folder.)
 ```
-docker run -v /absolute-path-to-this-folder/data/:/app/data:ro -v /absolute-path-to-this-folder/client.yaml:/app/client.yaml --network fedn_default mnist-client fedn run client -in client.yaml 
+docker run -v /absolute-path-to-this-folder/data/clients/0:/app/data:ro -v /absolute-path-to-this-folder/client.yaml:/app/client.yaml --network fedn_default mnist-client fedn run client -in client.yaml --name client0
 ```
-(Repeat above steps as needed to deploy additional clients).
+(Repeat above steps as needed to deploy additional clients but change the data partition (e.g. .../clients/1) and the --name).
 
-#### docker-compose
+### docker-compose
 To start 2 clients: 
 
 ```bash
-docker-compose -f docker-compose.yaml -f private-network.yaml up --scale client=2 
+docker-compose -f docker-compose.yaml -f private-network.yaml up 
 ```
+
 > If you are connecting to a Reducer part of a distributed setup or in Studio, you should omit 'private-network.yaml'. 
```

Comment:
Very minor here, just checking. Is this sentence correct? Shouldn't it be "**as** part of a distributed...."

Reply:
fixed

Reply:
Thanks!